### ISSUE TITLE
Bug: CMD + SHIFT + C - No longer works

### ISSUE THREAD

**Issue by CyberPunkCodes** [thomaslindstrom/color-picker#243]
I noticed the keyboard shortcut no longer works. I had an Atom update, so I updated it, restarted, uninstalled color-picker, re-installed color-picker, still doesn't work.

As of a couple atom versions ago, the color picker no longer works with `CMD + SHIFT + C`.

I can right click, and select `Color Picker` and the GUI pops up, but the keyboard shortcut doesn't work. I believe before I updated Atom (whatever version I was recently on) and re-installed color-picker (whatever version I was on), the right-click menu item wasn't showing either. Though after updating everything, it is there now. So that might have been fixed in the recent update? The keyboard shortcut wasn't fixed though.

Atom: 1.31.2
OS: OSX High Sierra 10.13.6

PS: Yes, it USED to work fine for me. I have no special key-bindings. Since this was known to work, I have even removed more plugins. So I don't believe this is a conflict from a new plugin, as I cleaned them up and have less now.

**EDIT:** Looks like the command has a conflict.

shift-cmd-C | code-format:format-code | Atom Ide Code Format | .platform-darwin atom-workspace atom-text-editor:not(.mini)

I think this is actually an Atom core plugin. So you should probably change the default keybinding, since out of the box, there is a conflict.

**Comment by LeaVerou** [comment]
Thank you! I run into the same issue, and your comment saved me some time :)

**Comment by thomaslindstrom** [comment]
Hi! Changing the default key binding is probably the way to go. Until then you can change the `color-picker` keybinding in its package settings:

<img width="796" alt="cleanshot 2018-10-15 at 13 09 25 2x" src="https://user-images.githubusercontent.com/2833514/46947587-8ecaa480-d07b-11e8-8b4f-1fab1e63d33d.png">

**Comment by CyberPunkCodes** [comment]
Yep. I think I changed mine to `H` representing in my mind `hex`.

**Comment by Glandalf** [comment]
Same problem for me. Could it be possible to propose an free text field to configure the "Trigger key"? Of course, it would be possible to hit some dumb thing, but also could let many options in case of shortcut conflicts. 
Nice job by the way!
